bin/check-alertclear-patterns.rb

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env ruby
+#
+# Check Alert/Clear Patterns plugin
```

Comment:
Lets use the standard header fomat outlined here: http://sensu-plugins.io/docs/developer_guidelines.html